README.md

```diff
@@ -1,5 +1,21 @@
 # vse-sync-pp
 
+The main purpose of this repo is to offer the packages necessary to run the reference implementation of synchronization tests.
+
+## Setup
+
+Installable Python kits, and information about using Python, are available at [python.org](python.org).
+
+1. [Install python >3.10](https://www.python.org/downloads/)
+1. [Install pip](https://packaging.python.org/en/latest/guides/installing-using-linux-tools/)
```

Comment:
While markdown will handle this properly, we don't always view it through a markdown viewer